resources/assets/js/app/views/topics/topicItem.js

```diff
@@ -1,7 +1,13 @@
 var Marionette = require('backbone.marionette');
+var moment = require('momentjs');
 
 module.exports = Marionette.ItemView.extend({
     template: 'topicItem',
     className: 'row post-item',
-    tagName: 'div'
+    tagName: 'div',
+    
+    initialize: function () {
+        this.model.attributes.created_at =
+            moment(this.model.attributes.created_at).format('dd.MM.YYYY');
```

Comment:
don't do this) if you need something in view, create additional key in serializeData and pass there formatted value like
{
  model: this.model.toJSON(),
  createdDate: moment(this.model.get('created_at')).format(...)